test/node/byte_utils.test.ts

```diff
@@ -465,6 +476,14 @@ describe('ByteUtils', () => {
         expect(bufferOut).to.be.an.instanceOf(Buffer);
         expect(bufferOut.buffer).to.equal(arrayBufferIn);
       });
+
+      it('should attempt fallback to Object.prototype.toString call if toStringTag does not exist on ArrayBuffer', () => {
```

Comment:
I would prefer we check this more directly via seeing if Object.prototype.toString is called